crates/starknet-devnet-core/src/starknet/get_class_impls.rs

```diff
@@ -118,6 +118,7 @@ mod tests {
 
         let block_number = starknet.get_latest_block().unwrap().block_number();
         let block_id = BlockId::Number(block_number.0);
+        starknet.create_block().unwrap();
```

Comment:
I spent some time trying to figure out why we need this line. Is my suggestion correct? If yes, we could keep it.
```suggestion
        starknet.create_block().unwrap(); // makes the queried block non-latest (and unsupported)
```

Reply:
Yes! Right, I'll add this comment to be more understandable.

Reply:
6dd8cbf95ca05849d8831989cf6d37fe152a9d4e